src/AzureDeprecation.APIs.REST/Startup.cs

```diff
@@ -1,12 +1,29 @@
 ﻿using AzureDeprecation.APIs.REST;
+using AzureDeprecation.APIs.REST.DataAccess.Repositories;
+using AzureDeprecation.APIs.REST.DataAccess.Contracts;
+using AzureDeprecation.APIs.REST.Settings;
 using AzureDeprecation.Runtimes.AzureFunctions;
 using Microsoft.Azure.Functions.Extensions.DependencyInjection;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
 
 [assembly: FunctionsStartup(typeof(Startup))]
-namespace AzureDeprecation.APIs.REST
+namespace AzureDeprecation.APIs.REST;
+
+public class Startup : DefaultStartup
 {
-    public class Startup : DefaultStartup
+    public override string ComponentName { get; } = "REST API";
+
+    protected override void ConfigureDependencies(IServiceCollection services)
     {
-        public override string ComponentName { get; } = "REST API";
+        base.ConfigureDependencies(services);
+
+        services.AddAutoMapper(typeof(Startup));
+        
+        services.AddOptions<CosmosDbOptions>().Configure<IConfiguration>((dbSettings, config) =>
+        {
+            config.GetSection(CosmosDbOptions.SectionName).Bind(dbSettings);
```

Comment:
This means our config will be as following, right?
```
{
  "cosmosDb": {
    "connectionString" : ""
  } 
}
```

To make this work, we need to update our deployment with this information. I can do that if you prefer but we'll need to hold off merging before this is done